doc/userguide/install.rst

```diff
@@ -140,34 +113,54 @@ one of the following ways::
 
 Minimal::
 
-    # Installed Rust and cargo as indicated above
```

Comment:
Why remove this part?

Reply:
Because there is no guide for Rust and Cargo install above this. It has been moved to the bottom of the guide in case `rustc` and `cargo` is not part of the package manager.

Reply:
Do we know if this is the case for any distros or OSs?  And could it be reduced to a single line, a link to Rust install page? This would most likely be a rare type install that we probably shouldn't try to cover.